css/sections/bento.css

```
@@ -0,0 +1,1047 @@
+	.custom-agents-container {
+		grid-template-columns: repeat(1, 1fr);
+	}
+	.bento_bentoImagesWrapper {
+		max-height: 309px;
+
+		display: flex;
+	}
+	.custom-agents-first-picture {
+		width: 50%;
+		margin-inline-end: 0;
+		margin-inline-start: 24px;
```

Stylelint (17.2.0)

[error] Expected shorthand property "margin-inline" (declaration-block-no-redundant-longhand-properties)

(declaration-block-no-redundant-longhand-properties)